scripts/themes-generator.js

```diff
@@ -0,0 +1,111 @@
+// themes-generator.js script performs several tasks:
+// - fetch data from Figma
+// - generate color library and create color-library json
+// - generate design tokens and create design-tokens json
+
+// Data received form Figma is a json file which has the following structure:
+// -'core' contains the colors library which defines the hex value for each color name, e.g.: 'tealscale-teal0': '#7fcbcd'
+// -'ide-default', 'ide-dark', ..., each contain two types of information:
+//    > mapping between color name and semantic value, e.g.: 'accent-strong': 'tealscale-teal0'
+//    > mapping between variable name of an element and the semantic value, e.g.: 'button-primary-bg-hover': 'accent-strong'
+
+
+// Data fetch from Figma is handled using the Figma API. As reported on the website (https://www.figma.com/developers/api)
+// "The Figma API supports read access and interactions with Figma files. This gives you the ability to view and extract any objects or layers, and their properties."

```

Comment:
Nice description. Let's put this into a README.md, with some info about how to run the script